drltrace/CMakeLists.txt

```diff
@@ -109,3 +109,12 @@ endif ()
 install(TARGETS drltrace DESTINATION "${INSTALL_BIN}"
   PERMISSIONS ${owner_access} OWNER_EXECUTE GROUP_READ GROUP_EXECUTE
   WORLD_READ WORLD_EXECUTE)
+
+##################################################
+# drltrace test
+
+if (BUILD_TOOL_TESTS)
+  get_target_property(app_path drsyscall_app LOCATION${location_suffix})
+  get_target_property(drltrace_path drltrace LOCATION${location_suffix})
+  add_test(drltrace ${drltrace_path} -- ${app_path})
```

Comment:
But I don't see any changes to the code, only a merge from master, so you could have waited for when you commit your code changes and it will rerun then, right?